go.mod

```diff
@@ -40,13 +40,13 @@ require (
 	github.com/hashicorp/go-version v1.3.0 // indirect
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
 	github.com/hashicorp/hcl v1.0.1-vault-3 // indirect
-	github.com/hashicorp/hcl/v2 v2.9.2-0.20210407182552-eb14f8319bdc // indirect
+	github.com/hashicorp/hcl/v2 v2.11.1 // indirect
```

Comment:
Without updating this dependency, the compile step should succeed as normal. I noticed that during my previous updates and didn't investigate yet. Otherwise, it looks good; thanks for maintaining the dependencies

Reply:
Thanks, I haven't looked at it yet either....

Reply:
Fine for me, it might be something we can investigate when resolving the issue with #51. Many (if not all) of our security alerts are related to dependencies and our dependency management.